doc/catalog.rst

```diff
@@ -0,0 +1,14 @@
+:mod:`eulxml.catalog` - Xml catalog for resolving schemas 
```

Comment:
This looks like it's probably reasonable, but I don't think you added it to the sphinx index, so it's not being built.